cli/src/pcluster/resources/head_node/user_data.sh

```diff
@@ -61,6 +61,12 @@ function vendor_cookbook
   export HOME="${!HOME_BAK}"
 }
 [ -f /etc/profile.d/proxy.sh ] && . /etc/profile.d/proxy.sh
+
+# deploy config files
+export PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin:/opt/aws/bin
+cfn-init -s ${AWS::StackName} -v -c deployFiles -r HeadNodeLaunchTemplate --region ${AWS::Region}
```

Comment:
what if this fails? Are we able to signal an error?

Reply:
later downstream, a `cd /tmp` is done before calling `cfn-init`. Is this a requirement that needs to be done also here?

Reply:
Move `/tmp` at the begining, we can't send the signal in deployFiles because we can't get wait condition ref, but if this command fails, creation will fail in 30 mins due to timeout of wait condition.